projects/laji/src/app/shared-modules/observation-result/observation-table/observation-table-settings.component.ts

```diff
@@ -9,9 +9,7 @@ import {ModalRef, ModalService} from 'projects/laji-ui/src/lib/modal/modal.servi
   selector: 'laji-observation-table-settings',
   template: `
```

Comment:
Not a fan of including the template in the ts-file, but probably it isn't a problem, and it was there before this PR anyway.

Reply:
yep i agree, i'll move it anyways